docs/install/build/browser/update-with-browser.md

```diff
@@ -1,12 +1,17 @@
 ## Update to Trio dev branch from Trio 0.2.x main
 
 !!! important "A Note on Compatibility"
-    Upgrading to Trio `v0.5.0` from `0.2.x` is smooth and straightforward. The new Onboarding Wizard will guide you step by step. Your pump, CGM, therapy settings and the last 24 hours of treatment and glucose history will be brought over automatically.
+    Upgrading to Trio `0.5.x` from `0.2.x` is smooth and straightforward. The new Onboarding Wizard will guide you step by step. Your pump, CGM, therapy settings and the last 24 hours of treatment and glucose history will be brought over automatically.
 
     **Be Aware**
     
-    * **Once you have upgraded to `v0.5.0`, going back to `0.2.x` is not supported**.
-    * If you choose to downgrade, you will need to set everything up again from scratch.
+    * **Once you have upgraded to `0.5.x`, going back to `0.2.x` is not supported**
+        * If you choose to downgrade, you will need to set everything up again from scratch
+    * Some guardrails may be more limiting that `0.2.x`
```

Comment:
Remove this bulletpoint please. It's an ongoing discussion, with HCPs and community members. This ought to not be part of docs to be removed later. We will not introduce the narrative of guardrails being a bad thing that needs removal into our docs.